system/ee/legacy/libraries/Typography.php

```diff
@@ -2597,11 +2597,11 @@ public function decodeIDN($url)
 		{
 			if (is_php('7.2'))
```

Comment:
I'm not spotting how it would fix it, but I may not be understanding.

Though we may not need it in the 'else'.  But I no longer have a great test environment.